libraries/botbuilder-core/etc/botbuilder-core.api.md

```diff
@@ -33,6 +33,7 @@ import { O365ConnectorCard } from 'botframework-schema';
 import { PasswordServiceClientCredentialFactory } from 'botframework-connector';
 import { ReceiptCard } from 'botframework-schema';
 import { ResourceResponse } from 'botframework-schema';
+import type { ServiceClientCredentials } from '@azure/ms-rest-js';
```

Comment:
This type should be exported from `botframework-connector` so we don't have to introduce `@azure/ms-rest-js` as a dependency in `botbuilder-core`.